app-drawer-layout/app-drawer-layout.html

```diff
@@ -156,7 +119,6 @@
           type: Boolean,
           readOnly: true,
           notify: true,
-          reflectToAttribute: true,
```

Comment:
When I first saw this I thought that that `blog-app` styling would not work because of `:not([attribute])` selectors not matching properly in IE/Edge, but for some weird reason (selector matches a class as well, maybe?) it does work.

I'll add the `reflectToAttribute` back to `narrow`.